mu/locale/es/LC_MESSAGES/mu.po

```diff
@@ -5,756 +5,1091 @@
+#: mu/modes/debugger.py:36
 msgid "Debug your Python 3 code."
-msgstr "DEBUG YOUR PYTHON 3 CODE."
+msgstr "Depura tu código de Python 3."
 
-#: mu/modes/debugger.py:48
+#: mu/modes/debugger.py:50 mu/modes/pygamezero.py:109 mu/modes/python3.py:166
 msgid "Stop"
-msgstr "STOP"
+msgstr "Detener"
 
-#: mu/modes/debugger.py:49
+#: mu/modes/debugger.py:51
 msgid "Stop the running code."
-msgstr "STOP RUNNING THE CODE"
+msgstr "Detener la ejecución del código."
 
-#: mu/modes/debugger.py:54
+#: mu/modes/debugger.py:57
 msgid "Continue"
-msgstr "CONTINUE"
+msgstr "Continuar"
 
-#: mu/modes/debugger.py:55
+#: mu/modes/debugger.py:58
 msgid "Continue to run your Python script."
-msgstr "CONTINUE TO RUN YOUR PYTHON SCRIPT."
+msgstr "Continúa ejecutando tu script de Python."
 
-#: mu/modes/debugger.py:60
+#: mu/modes/debugger.py:64
 msgid "Step Over"
-msgstr "STEP OVER"
+msgstr "Paso por encima"
 
-#: mu/modes/debugger.py:61
+#: mu/modes/debugger.py:65
 msgid "Step over a line of code."
-msgstr "STEP OVER A LINE OF CODE."
+msgstr "Paso sobre una línea de código."
 
-#: mu/modes/debugger.py:66
+#: mu/modes/debugger.py:71
 msgid "Step In"
-msgstr "STEP IN"
+msgstr "Paso dentro"
 
-#: mu/modes/debugger.py:67
+#: mu/modes/debugger.py:72
 msgid "Step into a function."
-msgstr "STEP INTO A FUNCTION."
+msgstr "Entrar en una función."
 
-#: mu/modes/debugger.py:72
+#: mu/modes/debugger.py:78
 msgid "Step Out"
-msgstr "STEP OUT"
+msgstr "Paso fuera"
 
-#: mu/modes/debugger.py:73
+#: mu/modes/debugger.py:79
 msgid "Step out of a function."
-msgstr "STEP OUT OF A FUNCTION."
+msgstr "Salir de una función."
 
-#: mu/modes/debugger.py:110
-msgid "Running script {}"
-msgstr "RUNNING SCRIPT {}"
-
-#: mu/modes/debugger.py:149
+#: mu/modes/debugger.py:156
 msgid "Your script has finished running."
-msgstr "YOUR SCRIPT HAS FINISHED RUNNING."
+msgstr "Tu script ha terminado de ejecutarse."
+
+#: mu/modes/debugger.py:224
+msgid ""
+"Unable to connect to the Python debugger.\n"
+"\n"
+msgstr ""
+"No ha sido posible conectar con el depurador de Python.\n"
+"\n"
 
-#: mu/modes/debugger.py:270
+#: mu/modes/debugger.py:308
 msgid "Debugger info: {}"
-msgstr "DEBUGGER INFO: {}"
+msgstr "Información del depurador: {}"
 
-#: mu/modes/debugger.py:276
+#: mu/modes/debugger.py:314
 msgid "Debugger warning: {}"
-msgstr "DEBUGGER WARNING: {}"
+msgstr "Advertencia del depurador: {}"
 
-#: mu/modes/debugger.py:283
+#: mu/modes/debugger.py:321
 msgid "Debugger error: {}"
-msgstr "DEBUGGER ERROR: {}"
+msgstr "Error del depurador: {}"
 
-#: mu/modes/microbit.py:34
+#: mu/modes/microbit.py:168
 msgid "BBC micro:bit"
-msgstr "BBC MICRO:BIT"
+msgstr "BBC micro:bit"
 
-#: mu/modes/microbit.py:35
+#: mu/modes/microbit.py:169
 msgid "Write MicroPython for the BBC micro:bit."
-msgstr "WRITE MICROPYTHON FOR THE BBC MICRO:BIT"
+msgstr "Escribe MicroPython para el BBC micro:bit."
 
-#: mu/modes/microbit.py:53
+#: mu/modes/microbit.py:190
+#, fuzzy
 msgid "Flash"
-msgstr "FLASH"
+msgstr "Flash"
```

Comment:
I think even in English "to flash" something is a term that most people, and some developers, would not know about, much less begginers/learners/teachers, is it the same in Spain?

`Flashear` sounds a bit too colloquial for me in this case (big button in the front and center of the UI), so if Flash works as well I think I prefer that one. However, it could probably be used as a verb in the other translations? Would  'Flashear' be better in something like the translation below?

```
msgid "Flash your code onto the micro:bit."
msgstr "Programa tu código en el micro:bit."
```
vs
```
msgstr "Flashea tu código en el micro:bit."
```

Because I've never used it in Spanish it sounds a bit wrong to me, but that's just my perception.
What do you think?

Reply:
I think it fine to use technical terms that people may not know... since Mu is a friendly place to introduce them. "What does Flash mean?" is answered in the docs and can easily be discovered via Google or simply asking your teacher. ;-) However, if there's a simpler and clearer way to say it than the technical term, I'd personally go for that.